File: ios/Runner/Plugins/osindicator/OsIndicatorPlugin.swift
import Flutter
import UIKit

/**
 * インジケータViewプラグイン　Factory登録を行う
 */
public class OsIndicatorPlugin: NSObject, FlutterPlugin {
    public static func register(with registrar: FlutterPluginRegistrar) {
        registrar.register(OsIndicatorFactory(messenger: registrar.messenger() as? (NSObject & FlutterBinaryMessenger)), withId: "local_plugins/activity_indicator")
    }
}

File: android/app/src/main/java/com/nineeyes/jp/flutter_ui_examples/platform_view_example/plugins/osindicator/OsIndicatorPlugin.java
package com.nineeyes.jp.flutter_ui_examples.platform_view_example.plugins.osindicator;

import io.flutter.plugin.common.PluginRegistry;

/**
 * インジケータViewプラグイン　Factory登録を行う
 */
public class OsIndicatorPlugin {
    public static void registerWith(PluginRegistry.Registrar registrar) {
        registrar
                .platformViewRegistry()
                .registerViewFactory(
                        "local_plugins/activity_indicator", new OsIndicatorFactory(registrar.messenger()));
    }
}